tests/test_to_container.py

```diff
@@ -187,14 +208,25 @@ def test_scmode(
             {"foo": 10, "bar": 10},
             id="dict_inter_listconfig",
         ),
+        param(
+            OmegaConf.create(
+                {"foo": ListConfig(content="${bar}"), "bar": ["a"]}
+            )._get_node("foo"),
+            "${bar}",
+            ["a"],
+            id="toplevel_list_inter",
+        ),
     ],
 )
 def test_to_container(src: Any, expected: Any, expected_with_resolve: Any) -> None:
     if expected is None:
         expected = src
     if expected_with_resolve is None:
         expected_with_resolve = expected
-    cfg = OmegaConf.create(src)
+    if isinstance(src, Container):
+        cfg = src
+    else:
+        cfg = OmegaConf.create(src)
```

Comment:
Would `cfg = _ensure_container(src)` work here?

Reply:
Yup, that works. Updated in https://github.com/omry/omegaconf/pull/730/commits/e67739babdb19837e7b7c877698f58e6bc4d25fb.